app/controllers/attendees_controller.rb

```diff
@@ -0,0 +1,56 @@
+class AttendeesController < ApplicationController
+  before_action :set_attendee, only: [:show, :edit, :update, :destroy]
+  before_action :set_event
+
+  def index
+    @attendees = @event.attendees
+  end
+
+  def new
+    @attendee = @event.attendees.build
+  end
+
+  def create
+    @attendee = @event.attendees.build(attendee_params)
+
+    if @attendee.save
+      redirect_to Event.find(@attendee.event_id)
+    else
+      render 'new'
+    end
+  end
+
+  def show
+  end
+
+  def edit
+  end
+
+  def update
+    if @attendee.update(attendee_params)
+      redirect_to event_attendees_path
```

Comment:
As we did in Sponsors, 
add parameter event_id to `event_attendees_path` to make it more clear.